### PR TITLE
[Rector] Enable TypedPropertyRector::INLINE_PUBLIC

### PR DESCRIPTION
**Pull request by samsonasik**
@MGatner this is to update not only private method, but also protected and public when possible.

### CODE REVIEW

**Review by MGatner — APPROVED** [review]
Thanks! Looks great.

File: src/Template/rector.php [hunk 1]
```diff
@@ -122,4 +123,8 @@
     $services->set(MakeInheritedMethodVisibilitySameAsParentRector::class);
     $services->set(SimplifyEmptyArrayCheckRector::class);
     $services->set(NormalizeNamespaceByPSR4ComposerAutoloadRector::class);
+    $services->set(TypedPropertyRector::class)
+        ->configure([
+            TypedPropertyRector::INLINE_PUBLIC => true,
```
Comment (kenjis) [inline src/Template/rector.php:128]: @samsonasik This should be `false`, shouldn't be?
https://github.com/rectorphp/rector/blob/a7f83696f3c453cfd52f42cf3b9142c1c94be8c1/rules/Php74/Rector/Property/TypedPropertyRector.php#L47

Comment (MGatner) [reply src/Template/rector.php:128]: We discussed this (I think PMs on Slack?) and the intent was that this Config file be used with projects, where developers can afford to change all their properties without BC concerns. 

Using this in libraries this rule should be changed, or it does create BC issues.

Comment (samsonasik) [reply src/Template/rector.php:128]: @kenjis it depends on what the usage. @MGatner we probably needs a note for it in the readme.

Comment (kenjis) [reply src/Template/rector.php:128]: I sent a PR #25

Comment (kenjis) [reply src/Template/rector.php:128]: @MGatner @samsonasik Thanks!

**Pull request merged**